src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java

```diff
@@ -415,7 +416,9 @@ public static String getDocumentRequest(CaseData caseData, String accessKey) thr
      */
     private static ReferralTypeData newReferralRequest(CaseData caseData) {
         return ReferralTypeData.builder()
-                .caseNumber(defaultIfEmpty(caseData.getEthosCaseReference(), null))
+                .caseNumber(defaultIfEmpty(StringUtils.isNotBlank(caseData.getMultipleReference())
+                        ? caseData.getMultipleReference()
+                        : caseData.getEthosCaseReference(), null))
```

Comment:
Would prefer StringUtils.isBlank() rather than isNotBlank(). Purely because semantically the 'else' part of this reads as NOT isNotBlank

Reply:
Refactored